### ISSUE TITLE
Familiarising ourselves with Git

### ISSUE THREAD

**Issue by ShaunDavid1** [PlaceholderGames/2016-Team2Thirds#5]
Each of us need to be have an understanding of how to use Git to be able to merge our work effectively.

**Comment by KieranClarkee** [comment]
I've found a tutorial explains how to use GitHub and Git well, which I found to be a good starting point in learning. Here is the link : https://www.youtube.com/watch?v=J_Clau1bYco

(When you reach the point where you Add, Commit and Push your files; Don't panic if it gives you a blank line as if nothing is happening and you've came into an error. It is because the file size is is large so just give it a few minutes and it will work.)

**Comment by ShaunDavid1** [comment]
Yeah we were just sorting out a master branch with Mike, because we were planning on using Connors work with multiplayer as our base.

**Comment by KieranClarkee** [comment]
Okay that sounds good, I need to remove the file I added. Could someone try help with that?

**Comment by ShaunDavid1** [comment]
What file are you on about? The master?

**Comment by KieranClarkee** [comment]
I have a "FirstPersonCamera" file up there that i'm trying to remove.

**Comment by Pagec531** [comment]
[Github.commands.cheat.sheet.txt](https://github.com/PlaceholderGames/Team2Thirds/files/562740/Github.commands.cheat.sheet.txt)

**Comment by Pagec531** [comment]
There is a little cheat sheet of git commands that we can use. I also think it might be a good idea to all meet up and just run over git in general? Unless of course everyone is ok as it is.

**Comment by Pagec531** [comment]
[Github.commands.cheat.sheet (4).txt](https://github.com/PlaceholderGames/Team2Thirds/files/716955/Github.commands.cheat.sheet.4.txt)

**Comment by Pagec531** [comment]
just added how to commit changes made.

**Comment by Pagec531** [comment]
[Github.commands.cheat.sheet.4.txt](https://github.com/PlaceholderGames/Team2Thirds/files/732880/Github.commands.cheat.sheet.4.txt)

**Comment by Pagec531** [comment]
[Github.commands.cheat.sheet.4.txt](https://github.com/PlaceholderGames/Team2Thirds/files/732884/Github.commands.cheat.sheet.4.txt)

**Comment by Pagec531** [comment]
The links there are just updated versions that helps out with merging our work 👍 just in case.
